wp-content/themes/cambodia-ict-camp/archive-facilitators.php

```diff
@@ -69,7 +68,7 @@
 
                                     get_template_part( 'inc/template-parts/facilitators/content', 'list' );
 
-                                    if( $counter%$wrap_count == 0 ) {
+                                    if( ($counter%$wrap_count == 0) || ($counter == $facilitator_group->post_count) ) {
```

Comment:
You need to add a condition of the if count equals the last index of loop or not. if it is the last, closed div should be added. For example, $wrap_count = 6; but posts have only 5?  5%6 =?